package.json

```diff
@@ -14,8 +14,12 @@
         "lint": "npm run lint:ci -- --fix",
         "lint:ci": "eslint .",
         "lint-styles": "stylelint 'src/**/*.s(a|c)ss'",
-        "test": "jest"
+        "test": "jest",
+        "prepublishOnly": "npm test && npm run build:prod"
```

Comment:
shouldn't check eslint and sasslint too?

Reply:
Fair point, added (while at it, I renamed `lint-styles` to `lint:styles` for consistency)